src/main/java/org/reaktivity/nukleus/socks/internal/stream/client/ConnectReplyStreamProcessor.java

```diff
@@ -0,0 +1,422 @@
+/**
+ * Copyright 2016-2017 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.socks.internal.stream.client;
+
+import static org.reaktivity.nukleus.buffer.BufferPool.NO_SLOT;
+
+import java.util.Optional;
+
+import org.agrona.DirectBuffer;
+import org.agrona.MutableDirectBuffer;
+import org.reaktivity.nukleus.function.MessageConsumer;
+import org.reaktivity.nukleus.socks.internal.metadata.State;
+import org.reaktivity.nukleus.socks.internal.stream.AbstractStreamProcessor;
+import org.reaktivity.nukleus.socks.internal.stream.Context;
+import org.reaktivity.nukleus.socks.internal.stream.Correlation;
+import org.reaktivity.nukleus.socks.internal.stream.types.SocksCommandResponseFW;
+import org.reaktivity.nukleus.socks.internal.stream.types.SocksNegotiationResponseFW;
+import org.reaktivity.nukleus.socks.internal.types.OctetsFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.AbortFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.BeginFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.DataFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.EndFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.ResetFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.WindowFW;
+
+final class ConnectReplyStreamProcessor extends AbstractStreamProcessor
```

Comment:
Why Processor?

Reply:
The idea would be that a Stream is the flow of data that is acted upon by this class. The bytes can be changed and data can be sent back or forwarded by the class. This was my intent in the naming.

Reply:
ok